dallinger/experiment.py

```diff
@@ -121,55 +166,12 @@ def organize_dashboard_tabs(cls, tabs):
         """A hook for custom organization of dashboard tabs in subclasses."""
         return tabs
 
-    def __init__(self, session=None):
-        """Create the experiment class. Sets the default value of attributes."""
-
-        #: Boolean, determines whether the experiment logs output when
-        #: running. Default is True.
-        self.verbose = True
-
-        #: String, the name of the experiment. Default is "Experiment
-        #: title".
-        self.task = "Experiment title"
```

Comment:
Can we add some documentation to this docstring to explain the intended use for `no_configure` and `configure`? Do we just have them for back-compatibility? Are we sure we want them long-term?

Reply:
It is a breaking change to change the signature of `Experiment.__init__` so I think we should be giving users some advice on how they should respond to it